src/fitnesse/responders/DisabledResponder.java

```diff
@@ -17,7 +17,7 @@ protected String contentFrom(FitNesseContext context, Request request, WikiPage
     return prepareResponseDocument(context).html(request);
   }
 
-  private HtmlPage prepareResponseDocument(FitNesseContext context) {
+  protected HtmlPage prepareResponseDocument(FitNesseContext context) {
```

Comment:
Why do you leave this method here (instead of just having the class use the new method you created in BasicResponder)?

Reply:
Because it has a different implementation. It doesn't uses the implementation of BasicResponder. I've added a `@Override` to make it clear.